test/sol5/kyberMatchingEngine.js

```diff
@@ -2199,7 +2199,6 @@ contract('KyberMatchingEngine', function(accounts) {
 
     describe("test edge cases for fee. very big / small network fee and very / big small custom fee", async() => {
         //todo: check what happens if fee combination is above 100% or equal 100% or very near 100%
```

Comment:
I think can remove this describe. no?
u already tested it?

Reply:
Fee combinations, no. Not yet

Reply:
wait, there was a line change here?

Reply:
Removed